src/main/java/de/tum/in/www1/artemis/service/compass/CompassCalculationEngine.java

```diff
@@ -262,44 +310,125 @@ public JsonObject getStatistics() {
         return jsonObject;
     }
 
-    private boolean hasConflict(int elementId) {
-        boolean conflict = false;
-        Assessment assessment = this.assessmentIndex.getAssessmentsMap().get(elementId);
-        if (assessment == null) {
-            return false;
-        }
-        for (List<Score> scores: this.assessmentIndex.getAssessmentsMap().get(elementId).getContextScoreList().values()) {
-            double uniqueScore = -1;
-            for (Score score: scores) {
-                if (uniqueScore != -1 && uniqueScore != score.getPoints()) {
-                    conflict = true;
+
```

Comment:
the name of the method does not really fit to is implementation. Please add some comments and think about renaming the method name

Reply:
Yep this bothered me too.. I think the best option would be to completly remove the method and dissolve it in notifyNewAssessment an addNewManualAssessment